src/module/number_guess.py

```
@@ -0,0 +1,100 @@
+        if not check_is_int(participant_guess_plain):
+            message.reply("参数格式错误，请输入数字")
+            return None
+
+        participant_guess = int(participant_guess_plain)
+
+        current_info: GuessInfo = _guess_info[current_uuid]
+        if participant_guess == current_info.target:
+            message.reply(f"恭喜你猜对了，答案是 {participant_guess}，总共猜了 {current_info.trials} 次")
+            current_info = GuessInfo(GuessStatus.ENDED, -1, (-1, -1), -1)
+        elif participant_guess > current_info.target:
+            current_info.bound = (current_info.bound[0], min(participant_guess - 1, current_info.bound[1]))
```

Pylint (3.3.7)

[convention] Line too long (107/100)

(C0301)